examples/dt/uni-alpha/control-plane/service-values.yaml

```diff
@@ -0,0 +1,138 @@
+---
+apiVersion: v1
+kind: ConfigMap
+
+metadata:
+  name: service-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+
+data:
+  cinderVolumes:
+    customServiceConfig: |
+      [DEFAULT]
+      enabled_backends = lvm
+
+      [lvm]
```

Comment:
See https://github.com/openstack-k8s-operators/cinder-operator/blob/main/config/samples/backends/lvm/iscsi/backend.yaml#L21 for the latest updates.

Specifically, you now need to configure the target_ip_address=172.18.0.X and target_secondary_ip_addresses = 172.19.0.Y addresses to match the OCP node with the "openstack.org/cinder-lvm=" label. These can be found in your uni-alpha/control-plane/nncp/values.yaml file.

Reply:
Done. Kindly review the rendered output.